packages/flutter/lib/src/rendering/custom_layout.dart

```diff
@@ -127,9 +127,11 @@ abstract class MultiChildLayoutDelegate {
 
   /// True if a non-null LayoutChild was provided for the specified id.
   ///
-  /// Call this from the [performLayout] or [getSize] methods to
-  /// determine which children are available, if the child list might
-  /// vary.
+  /// Call this from the [performLayout] method to determine which children
+  /// are available, if the child list might vary.
+  ///
+  /// Avoid calling this from [getSize] as the child list is not available
+  /// until layout size is determined.
```

Comment:
"Avoid" is probably to weak here, this crashes if you call it from getSize, right? Maybe better:

"This method cannot be called from [getSize] as the size is not allowed to depend on the children" or something like that.